### PR TITLE
Issue #26 intro to deep learning

### LINKED ISSUE #26: Intro to Deep Learning
Use TensorFlow and Keras to build and train neural networks for structured data.
- [x] A Single Neuron
- [x] Deep Neural Networks
- [x] Stochastic Gradient Descent
- [x] Overfitting and Underfitting
- [x] Dropout and Batch Normalization
- [x] Binary Classification
- [x] Bonus 1: Detecting the Higgs Boson With TPUs

### PR DESCRIPTION
**Pull request by drakearch**
Closes #26

### CODE REVIEW

**Review by drakearch — COMMENTED** [review]
💯
